mountpoint-s3-client/examples/client_benchmark.rs

```diff
@@ -116,11 +117,11 @@ fn run_benchmark(
         let received_size = received_size.load(Ordering::SeqCst);
         total_bytes += received_size;
         println!(
-            "{}: received {} bytes in {:.2}s: {:.2} Gib/s",
+            "{}: received {} bytes in {:.2}s: {:.2} Gb/s",
             iteration,
             received_size,
             elapsed.as_secs_f64(),
-            (received_size as f64) / elapsed.as_secs_f64() / (1024 * 1024 * 1024 / 8) as f64
+            (received_size as f64) / elapsed.as_secs_f64() / (1000 * 1000 * 1000 / 8) as f64
```

Comment:
Mentioned this in another Python PR - perhaps we should make this a function that can be reused to make sure the logic is clear and correct in all cases